### PR TITLE
[APPS-3459] Update en.yml file with marketplace listing content

### PR DESCRIPTION
**Pull request by pmgrove**
:v:

/cc @zendesk/vegemite, @zendesk/i18n, @zendesk/platypus 

### Description

Update en.yml file with marketplace listing content. Even though this app is hidden on the marketplace we should still make this update to ensure the installation instructions are still present on the manage page.

### References
* JIRA: https://zendesk.atlassian.net/browse/APPS-3459

### Risks
* [low] - app content may not show on marketplace

### CODE REVIEW

**Comment by pmgrove** [comment]
@ydrain, this app has no package in Rosetta. Can we add one?

**Comment by ydrain** [comment]
Please add one as needed.

File: translations/en.yml [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+3. Configure the settings in Microsoft Dynamics CRM to view Microsoft Dynamics CRM 2011 data in Zendesk Support.
+
+4. Configure the “Zendesk settings” page in Microsoft Dynamics to set up user credentials for the Zendesk Support account you want to connect to and data you wish to see in Zendesk.
+
+  * (Optional) Configure the ticket-to-case capabilities in Zendesk Support and Microsoft Dynamics CRM to convert tickets into Microsoft Dynamics CRM cases.
+
+5. Add the Zendesk Support ticketing grid to any of the available entities in Microsoft Dynamics CRM.
+
+
+For complete setup instructions, see the [Microsoft Dynamics CRM for Zendesk Getting Started guide](https://support.zendesk.com/entries/33898297-Zendesk-for-Microsoft-Dynamics-CRM-Part-1-Installing-the-Zendesk-for-Microsoft-Dynamics-CRM-as-a-mod)."
```
Comment (ydrain) [inline translations/en.yml]: Update URL?

File: translations/en.yml [hunk 2]
```diff
@@ -0,0 +1,107 @@
+---
+title: "ms_dynamics_app"
+packages:
+  - default
+  - app_ms_dynamics_app
+
+parts:
+  - translation:
+      key: "txt.apps.ms_dynamics_app.app.name"
+      title: "Name for App: Microsoft"
```
Comment (injuhwang) [inline translations/en.yml]: Please add the following:
> Do not translate \"Microsoft.\"

File: translations/en.yml [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+      value: "Microsoft"
+  - translation:
+      key: "txt.apps.ms_dynamics_app.app.short_description"
+      title: "Description for App: Microsoft"
```
Comment (injuhwang) [inline translations/en.yml]: Please add the following:
> Do not translate \"Microsoft Dynamics.\"

File: translations/en.yml [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+      value: "Show custom Microsoft Dynamics data about the ticket requester"
+  - translation:
+      key: "txt.apps.ms_dynamics_app.app.long_description"
+      title: "Long Description for App: Microsoft"
```
Comment (injuhwang) [inline translations/en.yml]: Please add the following:
> Do not translate \"Microsoft Dynamics.\"

File: translations/en.yml [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+
+* Improve productivity and workflow efficiencies by applying Microsoft Dynamics CRM reporting and analytics tools to Zendesk Support ticket data.
+
+* Measure customer satisfaction and trends to gain a complete view of your Support and Sales team operations.
+
+
+By enabling this app, You agree to the [Built by Zendesk Terms of Use](https://www.zendesk.com/company/built-by-zendesk-agmt/)."
+  - translation:
+      key: "txt.apps.ms_dynamics_app.app.installation_instructions"
+      title: "Installation Instructions for App: Microsoft"
```
Comment (injuhwang) [inline translations/en.yml]: Please add the following:
> Do not translate \"Microsoft Dynamics.\"

File: translations/en.yml (hunk 1 above)
Comment (injuhwang) [inline translations/en.yml]: @pmgrove, Please replace the URL with this: https://support.zendesk.com/hc/en-us/articles/203660156
It'll help us to track down in the future when we need to edit it.

File: translations/en.yml [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+title: "ms_dynamics_app"
+packages:
+  - default
+  - app_ms_dynamics_app
+
+parts:
+  - translation:
+      key: "txt.apps.ms_dynamics_app.app.name"
+      title: "Name for App: Microsoft"
+      value: "Microsoft"
```
Comment (injuhwang) [inline translations/en.yml]: Please add `hidden: true`.

**Comment by pmgrove** [comment]
@injuhwang 💇

**Comment by injuhwang** [comment]
👍 for the strings

**Pull request merged**